mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/ContainerExtractor.java

```diff
@@ -19,7 +21,9 @@
 @Retention(RetentionPolicy.RUNTIME)
 public @interface ContainerExtractor {
 
-	Class<? extends org.hibernate.search.mapper.pojo.extractor.ContainerExtractor> type();
+	BuiltinContainerExtractor value() default BuiltinContainerExtractor.AUTOMATIC;
+
+	Class<? extends org.hibernate.search.mapper.pojo.extractor.ContainerExtractor> type() default DefaultExtractors.class;
```

Comment:
Ok I'll use `UndefinedContainerExtractorImplementationType` as default, because the `DefaultExtractors` will be eventually replaced by `BuiltinContainerExtractor.AUTOMATIC`, won't it? May I add a comment?

Reply:
Maybe just do it all in the same commit? Make sure that `AUTOMATIC` is interpreted as "automatically resolve extractors", remove `DefaultExtractors` and introduce `UndefinedContainerExtractorImplementationType` as a default here?

Reply:
@yrodiere I'm not sure I've got it. I did this: https://github.com/hibernate/hibernate-search/blob/af858e6c61913f8fd97bae9d86bce44f1478cfea/mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/ContainerExtractorRef.java.
Is it what you meant? Thanks

Reply:
That's a start, yes.

But as I said (twice):

> Once it's done, you can remove the DefaultExtractors class.

> remove DefaultExtractors

Normally `DefaultExtractors` is not needed anymore:

* other annotations using it as a default value can use `@ContainerExtractor(BuiltinContainerExtractor.AUTOMATIC)` instead
* there shouldn't be any explicit reference to it elsewhere in the tests
* the annotation parsing code I gave you should use `UndefinedContainerExtractorImplementationType` instead of `DefaultExtractors`.

Reply:
Ok thanks